redfish-core/lib/pcie.hpp

```diff
@@ -712,7 +712,7 @@ inline void requestRoutesSystemPCIeDevice(App& app)
             [](const crow::Request& req,
                const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                const std::string& device) {
-        std::optional<std::vector<nlohmann::json>> slotsData;
+        std::optional<nlohmann::json> slotsData;
```

Comment:
would be good to call this oem / oemData / something other than slotsData. E.g. https://github.com/ibm-openbmc/bmcweb/blob/07faf31c26909fb3cdcfc77f82c682bd58b50cb3/redfish-core/lib/managers.hpp#L2315

Reply:
> would be good to call this oem / oemData / something other than slotsData. E.g.
> 
> https://github.com/ibm-openbmc/bmcweb/blob/07faf31c26909fb3cdcfc77f82c682bd58b50cb3/redfish-core/lib/managers.hpp#L2315

Changed.